components/dashboard/src/teams/NewTeam.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { FormEvent, useContext, useState } from "react";
+import { useHistory } from "react-router-dom";
+import { getGitpodService } from "../service/service";
+import { UserContext } from "../user-context";
+
+export default function () {
+    const { user, setContext } = useContext(UserContext);
+    const history = useHistory();
+
+    const [ creationError, setCreationError ] = useState<Error>();
+    let name = '';
+    const createTeam = async (event: FormEvent) => {
+        event.preventDefault();
+        try {
+            if (!user) {
+                throw new Error('Not signed in');
+            }
+            const team = await getGitpodService().server.createTeam(name);
+            const teams = await getGitpodService().server.getTeams();
+            setContext({ user, teams });
+            history.push(`/${team.slug}`);
+        } catch (error) {
+            console.error(error);
+            setCreationError(error);
+        }
+    }
+    return <div className="flex flex-col w-96 mt-16 mx-auto items-center">
+        <h1>New Team</h1>
+        <p className="text-gray-500 text-center text-base">Teams allow you to <strong>group multiple projects</strong>, <strong>collaborate with others</strong>, <strong>manage subscriptions</strong> with one centralized billing, and more. <a href="">Learn more</a></p>
```

Comment:
**issue**: Reminder to use the same style (underline) if possible. Alternatively, a blue color for the link could suffice for making this slightly more accessible.